Source/Charts/Renderers/BarChartRenderer.swift

```diff
@@ -399,9 +396,109 @@ open class BarChartRenderer: BarLineScatterCandleBubbleRenderer
                 context.fill(barRect)
             }
         }
-        
+
+        if dataSet.drawBarGradientEnabled {
+            drawGradientBars(context: context, dataSet: dataSet, buffer: buffer, matrix: valueToPixelMatrix)
+        } else {
+            drawDefaultBars(context: context, dataSet: dataSet, dateSetIndex: index, buffer: buffer)
```

Comment:
maybe drawNormalBars or standard bars, etc? not sure which is better. @jjatie idea?